validator/client/sync_committee.go

```diff
@@ -51,11 +53,22 @@ func (v *validator) SubmitSyncCommitteeMessage(ctx context.Context, slot types.S
 		log.WithError(err).Error("Could not get sync committee message signing root")
 		return
 	}
+	fork, err := forks.Fork(slots.ToEpoch(slot))
+	if err != nil {
+		log.WithError(err).Errorf("Could not get fork on current slot: %d", slot)
+		return
+	}
```

Comment:
We should be able to move this piece of logic inside of the actual web3signer keymanager Sign function, as it's the only one that needs that